pinot-minion/src/main/java/org/apache/pinot/minion/BaseMinionStarter.java

```diff
@@ -116,6 +117,7 @@ public void init(PinotConfiguration config)
       _instanceId = CommonConstants.Helix.PREFIX_OF_MINION_INSTANCE + _hostname + "_" + _port;
     }
     _listenerConfigs = ListenerConfigUtil.buildMinionConfigs(_config);
+    _tlsPort = ListenerConfigUtil.findLastTlsPort(_listenerConfigs, 0);
```

Comment:
this should be -1

Reply:
This is 0 for other participant types, shall we change for them as well?

Reply:
hmm, you mean all other components generating tls port for admin apis if not specified?

I checked broker is using -1.